cobra/flux_analysis/find_active_reactions.py

```diff
@@ -0,0 +1,602 @@
+
+"""
+Find all active reactions by solving a single MILP problem
+or a (small) number of LP problems
+
+"""
+import logging
+from cobra.flux_analysis.variability import flux_variability_analysis
+from cobra.flux_analysis.helpers import normalize_cutoff
+from optlang import Model, Variable, Constraint, Objective
+from optlang.symbolics import Zero
+from scipy.linalg import orth
+import numpy as np
+
+LOGGER = logging.getLogger(__name__)
+
+
+def relax_model_bounds(model, bigM=1e4):
+    """
+    Relax all upper and lower bounds in the model.
+    All positive upper bounds will become bigM.
+    All negative lower bounds will become -bigM.
+    All positive lower bounds and negative upper bounds will become zero.
+
+    Parameters
+    ----------
+    model: cobra.Model
+        cobra model. It will *not* be modified.
```

Comment:
You say the model is not modified but this function changes the bounds on every reaction so that seems a little misleading.

Reply:
Good catch. Just copied and pasted. Fixed.